presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -269,4 +271,16 @@ public FeaturesConfig setRegexLibrary(RegexLibrary regexLibrary)
         this.regexLibrary = regexLibrary;
         return this;
     }
+
+    public boolean isCharPadSpaces()
+    {
+        return charPadSpaces;
+    }
+
+    @Config("char.pad-spaces")
```

Comment:
I will move commit introducing `PAD SPACES` flag to the end, so that previous commits can be merged quicker